File: 26. Remove Duplicates from Sorted Array/RemoveDuplicatesFromSortedArray.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        
        int duplicates = 0;

        for (int i = 1; i < nums.Length; ++i) {

            if (nums[i] == nums[i-1]) {
                duplicates++;
            }
            else {
                nums[i-duplicates] = nums[i];
            }
        }
        return nums.Length-duplicates;
    }
}

File: 26. Remove Duplicates from Sorted Array/RemoveDuplicatesFromSortedArray.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        duplicates = 0

        for index, num in enumerate(nums[1:],1):

            if num == nums[index-1]:
                duplicates += 1
            else:
                nums[index-duplicates] = num

        return len(nums)-duplicates
